client/js/assets/fonts/LICENSE.txt

```diff
@@ -0,0 +1,202 @@
+

```

Comment:
Do you need this?

Reply:
Yes, it is because these where paid fonts apparently so the licence is included